dotcom-rendering/src/components/Picture.tsx

```diff
@@ -39,11 +43,41 @@ const decideImageWidths = ({
 	role,
 	isMainMedia,
 	format,
+	isLightbox,
+	orientation,
 }: {
 	role: RoleType;
 	isMainMedia?: boolean;
 	format: ArticleFormat;
+	isLightbox: boolean;
+	orientation: Orientation;
 }): [ImageWidthType, ...ImageWidthType[]] => {
+	if (isLightbox) {
+		switch (orientation) {
+			case 'portrait':
+				return [
+					{ breakpoint: breakpoints.mobile, width: 480 },
+					{ breakpoint: breakpoints.mobileLandscape, width: 480 },
+					{ breakpoint: breakpoints.phablet, width: 480 },
+					{ breakpoint: breakpoints.tablet, width: 660 },
+					{ breakpoint: breakpoints.desktop, width: 740 },
+					{ breakpoint: breakpoints.leftCol, width: 980 },
+					{ breakpoint: breakpoints.wide, width: 1140 },
+				];
+
+			case 'landscape':
+			default:
```

Comment:
We can avoid the default branch by making orientation have a default value of `'landscape'`.

Reply:
Why was this resolved? The default case is not necessary as the switch is exhaustive.